src/nl/surf/eduhub_rio_mapper/rio/loader.clj

```diff
@@ -148,6 +149,9 @@
             xml (soap/prepare-soap-call (str "opvragen_" type) rio-sexp (make-datamap institution-oin recipient-oin) credentials institution-oin recipient-oin)]
         (assert (not (errors? xml)) "unexpected error in request body")
         (handle-opvragen-request type
+                                 (if xml-response rio-xml-getter-response
+                                                  (case type "opleidingsrelatiesBijOpleidingseenheid" rio-relation-getter-response
+                                                             rio-getter-response))
```

Comment:
So we're not supporting getting XML for a "opleidinsrelatiesBijOpliedingseenheid"?  If not, please put in a comment here, to explain why.